ethereumj-core/build.gradle

```diff
@@ -4,17 +4,19 @@ buildscript {
     repositories {
         jcenter()
         maven { url 'http://repo.spring.io/plugins-release-local' }
+        maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
         classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
         classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:3.0.1'
+        classpath "com.github.jengelman.gradle.plugins:shadow:1.2.3"
     }
 }
 
 plugins {
     id 'application'
     id 'jacoco'
-    id 'com.github.johnrengelman.shadow' version '1.2.0'
+    id 'com.github.johnrengelman.shadow' version '1.2.3'
```

Comment:
Just checked again on a fresh clone, applying changes to this line only does work for me. Version change is enough